ihmc-java-toolkit/src/main/java/us/ihmc/tools/ArrayTools.java

```diff
@@ -32,7 +32,7 @@ public static double[] parseDoubleArray(String stringSource) throws IOException
       StringTokenizer tokenizer = new StringTokenizer(line.replace("{", "").replace("}", "").replace(" ", "").replace(";", ""), ",");
       while (tokenizer.hasMoreElements())
       {
-         retArray.add(new Double(Double.parseDouble((String) tokenizer.nextElement())));
+         retArray.add(Double.valueOf(Double.parseDouble((String) tokenizer.nextElement())));
```

Comment:
I would imagine this would be sufficient?
```suggestion
         retArray.add(Double.parseDouble((String) tokenizer.nextElement()));
```

Reply:
I'll probably shelve this PR and do anotehr pass on cleaning up deprecations. They make the build log a pain.